CHANGELOG.md

```diff
@@ -33,12 +33,16 @@ This release is built on top of [OpenTelemetry .NET](https://github.com/open-tel
   Both supported only on .NET Framework.
 - `WCFCLIENT` and `WCFSERVICE` no longer requires manual modification
   of config files to include `TelemetryEndpointBehaviorExtensionElement`.
+- [GraphQL](https://www.nuget.org/packages/GraphQL/) instrumentation is now
+  supported from version 7.5.0.
```

Comment:
GraphQL.NET 7.5 only supports OpenTelemetry on .NET 5+ because the necessary APIs do not exist within .NET Framework or .NET Standard.